src/lib/progress-bar/progress-bar.spec.ts

```diff
@@ -118,6 +120,53 @@ describe('MatProgressBar', () => {
       expect(progressElement.componentInstance.mode).toBe('buffer');
     });
   });
+
+  describe('animation trigger on determinate setting', () => {
```

Comment:
It would be good to also have a unit test with `NoopAnimationsModule`

Reply:
I'm not completely following the test where NoopAnimationsModule here, would it look like:
1. Check that event is not bound?
2. On animation === noop, always call output on value change? 

Or would it be something else?

Reply:
I want to ensure that we have a test that covers the code path where `NoopAnimationsModule` is loaded and someone tries to register this event. The goal being to ensure that no error is thrown more so than that a particular outcome occurs.

Reply:
Thanks that makes sense. I added some more test with NoopAnimations module turned on. Also I make the animationEnd fire when setting the value in NoopAnimations mode, my thoughts is that page author might be relying on this output to update their DOM in their test so we should fire the output synchronously when there's no animation. PTAL.